pastel/client.go

```diff
@@ -90,7 +90,7 @@ func (client *client) FindTicketByID(ctx context.Context, pastelID string) (*IDT
 // TicketOwnership implements pastel.Client.TicketOwnership
 func (client *client) TicketOwnership(ctx context.Context, txID, pastelID, passphrase string) (string, error) {
 	var ownership struct {
-		Art   string `json:"art"`   // txid from the request
+		NFT   string `json:"NFT"`   // txid from the request
```

Comment:
its fine, but `json:"NFT"` should be json:"nft"
or maybe confirm with ayk to fix it in cnode : https://github.com/pastelnetwork/pastel/blob/7a2908a535e187194c224ed7819872d1d2594930/src/mnode/mnode-rpc.cpp#L2672 or not.